### ISSUE TITLE
goldHEN automatically loaded after reboot!!!

### ISSUE THREAD

**Issue by carrey2018** [GoldHEN/GoldHEN#160]
Last time when I was trying to pppwn the console, it was a little laggy, so I tried to dial again via pppoe. Yet it immediately crashed and rebooted. After a disc check, I found goldHEN was loaded without pppwn! I think there must be a way of keeping goldhen fixed in the system even after reboot although I don't know how this happened.

**Comment by Masamune3210** [comment]
Unfortunately, you haven't stumbled onto some weird exploit that lets you run GoldHEN without doing anything beforehand, you have just experience something that isn't really well explained. Basically, part of the system crashed and caused something called a soft reboot, the system never actually turned all the way off and back on, but much of the userland (the parts that you see and interact with) were restarted, making it seem as though the system rebooted and had the exploit from nowhere


TL;DR: Exploit succeeded but something went wacky either during or afterwards, triggering a soft reboot. Unfortunately, not a way to load payloads without an exploit. Sorry to burst your bubble.
